docs/pom.xml

```diff
@@ -40,11 +40,96 @@
                         </goals>
                         <configuration>
                             <target>
-                                <exec dir="${project.basedir}" executable="jekyll" failonerror="true">
+                                <available file="${project.build.directory}/che-repo" property="cheRepoExists" type="dir"/>
```

Comment:
Binding external docker daemons is something I would like to look into next quarter. There are many users that want to use docker in the workspace machine. It would be part of the CLI like che-mount utility. Most docker daemons use docker.sock though which I cannot mount inside a workspace machine. This requires some investigation. 

I would like to actually end up with the host providing their own external docker daemon automatically. Either secure a dind container or provide a VM with docker daemon. This requires additional investigation though.